src/Rox/Core/Roxx/Parser.php

```diff
@@ -230,6 +242,24 @@ function (ParserInterface $parser, StackInterface $stack, ContextInterface $cont
             $stack->push(base64_decode($op1));
         });
 
+        $this->addOperator("tsToNum", function (ParserInterface $parser, StackInterface $stack, ContextInterface $context) {
+            $op1 = $stack->pop();
+
+            /**
+             * For some reason, get_class, or instanceof, does not check properly the type assertion;
+             * Adding this workaround to make sure that it behaves as an DateTime instance;
+             */
```

Comment:
if you think that's best, that fine, but it works for me:
```
$now = new DateTime();
// or @now = new DateTimeImmutable();
$isADate = $now instanceof DateTimeInterface
echo $isADate
```
results: (1)
otherwise ($now='string') return empty